### PR TITLE
Feature/wf 583 product description

### PR DESCRIPTION
**Pull request by geroldboehler**
## Pull request
Adds product description to c-product-detail component.
 
### Ticket
https://winterhalter-fenner.atlassian.net/browse/WF-583
 
### Browser testing
http://localhost:8080/styleguide/product-detail
 
### Checklist
- [x] Added JSDoc and styleguide demo
- [x] Tested all links in project relevant browsers
- [x] Tested all links in different screen sizes
- [x] Did run automated tests and linters
- [x] Did assign ticket
- [x] Double checked target branch
 
## Review/Test checklist
- [x] Did run automated tests and linters
- [x] Did review code and documentation
- [x] Tested all links in project relevant browsers
- [x] Tested all links in different screen sizes
- [x] Re-assign ticket to developer

### CODE REVIEW

File: app/components/c-product-detail.vue [hunk 1]
```diff
@@ -177,6 +180,26 @@
...
+        @include media(sm) {
+          padding-left: $spacing--30;
+        }
+      }
+    }
+
+    &__description {
+      @include font($font-size--14, $line-height: 18px);
+
+      padding: $spacing--10 $spacing--20 $spacing--10 $spacing--20;
```
Comment (claudio147) [inline app/components/c-product-detail.vue]: use `padding: $spacing--10 $spacing--20`, it's the same result

File: app/components/c-product-detail.vue [hunk 2]
```diff
@@ -177,6 +180,26 @@
...
+
+        @include media(sm) {
+          padding-left: $spacing--30;
+        }
+      }
+    }
+
+    &__description {
+      @include font($font-size--14, $line-height: 18px);
+
```
Comment (claudio147) [inline app/components/c-product-detail.vue]: color is wrong should be `$color-grayscale--200`now it's `#212529` (from body)

**Comment by claudio147** [comment]
Are you sure that you want to use a single component for `c-product-detail` and implement all the inside components inside? Or what's the plan?

File: app/components/c-product-detail.vue [hunk 3]
```diff
@@ -26,7 +26,10 @@
     <section :class="b('bottom')">
 
       <div :class="b('main', { area: 'bottom' })">
-        <div :class="b('details')"> details</div>
+        <div v-if="product.description" :class="b('details')">
```
Comment (claudio147) [inline app/components/c-product-detail.vue]: - [ ] As I understand the `details` section is not only for `product.description` There are also the accordions with "technical details", documents etc. So I would use the `details` as a wrapper for this components

File: app/components/c-product-detail.vue [hunk 4]
```diff
@@ -26,7 +26,10 @@
...
+          <e-heading underline tag-name="h2" color="gray">{{ $t('c-product-detail.product-description') }}</e-heading>
```
Comment (claudio147) [inline app/components/c-product-detail.vue]: - [ ] Would name the translation `c-product-detail.product-description-title`
How many different heading colors we have? If there are more than 2 I would recommend to use here color codes

**Comment by patric-eberle** [comment]
@claudio147 I think for the description text this is OK. But anything more sophisticated should be a separate component.

File: app/components/c-product-detail.vue [hunk 5]
```diff
@@ -26,7 +26,10 @@
...
+          <div :class="b('description')" v-html="product.description"></div>
+        </div>
```
Comment (claudio147) [inline app/components/c-product-detail.vue]: General: How is the concept about the product-detail? Which components are separate, single components which are integrated in the `c-product-detail` and which are directly implemented here?

Comment (silvanbaumann) [reply app/components/c-product-detail.vue]: @claudio147 @geroldboehler Hey guys, just picked-up this conversation about the product detail. As Patric explained to me, this component is only a wrapper for all the other components e.g. gallery, accordion etc. As you can see I already tried to name the different sections. Let's talk with Patric about that.

**Pull request merged**